docs/docs/start/tree-start.mdx

```diff
@@ -23,7 +23,7 @@ npm install fluid-framework@latest
 A `SharedTree` can be created by defining a `ContainerSchema` with an initial object of type `SharedTree` and using this schema to create and load your container.
 This example creates a container using an Azure specific client.
 
-**note**: `enableRuntimeIdCompressor` must be enabled in the <ApiLink packageName="container-runtime" apiName="IContainerRuntimeOptions" apiType="interface">container runtime options</ApiLink> in order to use `SharedTree`
+**note**: `enableRuntimeIdCompressor` must be enabled in the <ApiLink packageName="container-runtime" apiName="IContainerRuntimeOptions" apiType="TypeAlias">container runtime options</ApiLink> in order to use `SharedTree`
```

Comment:
PR #24427 converted `IContainerRuntimeOptions` from an `interface` to a `type`. File names for API docs currently include the type of API item being represented, so this code change resulted in a new file for the API item. Our "smart links" (e.g. `ApiLink`) don't currently have enough context to automatically determine this context, so this link was broken by the change.

Longer term, we should invest some time into making `ApiLink` a bit smarter in this regard, so that users don't have to explicitly specify the API item type. But for now, changing from "Interface" to "TypeAlias" fixes the broken link.